backend/data_preparation/crawler/twitter_crawler.py

```diff
@@ -34,7 +34,8 @@ def __init__(self, extractor: ExtractorBase = None, dumper: DumperBase = None):
         self.crawled_id_set: Set[int] = set()
         self.data_from_db_count = 0
 
-    def start(self, keywords: List[str], batch_number: int, fetch_from_db: bool = False, end_clause=None) -> None:
```

Comment:
sorry one thing I forgot in this. 

The `start` method is deprecated in crawler. it is replaced by `Runnable.run()` method. 

this method is left here for testing purpose. 

Please make the corresponding change to `tasks/TextFromTweetRunnable.py`.

and test it with `task_manager.py`.

after the test, you may safely remove the `start()` method here. (leave a empty definition, will be removed with the base class).